scripts/cat/cats.py

```diff
@@ -819,79 +824,67 @@ def add_to_clan(self) -> list:
         for child_id in children:
             child = Cat.all_cats[child_id]
             if (
-                child.outside
-                and not child.exiled
-                and not child.dead
-                and child.moons < 12
+                    child.outside
+                    and not child.exiled
+                    and not child.dead
+                    and child.moons < 12
             ):
                 child.add_to_clan()
                 History.add_beginning(child)
                 ids.append(child_id)
 
         return ids
 
-    def status_change(self, new_status, resort=False):
+    def status_change(self, new_status: StatusEnum | str, resort=False):
```

Comment:
We should not offer/encourage passing strings instead of their string enum counterpart.

Reply:
I wholeheartedly agree - except that it means that a lot of classes need StatusEnum as a dependency now in order to assign one. If small class dependencies isn't an issue, I'll go through and change it (I think 99% of things have it there anyway)

Reply:
I think the benefits of having clear, refactoring safe enums in place of magic strings everywhere possible reduces our headaches overall, so feel free to go ahead.